src/adv.py

```diff
@@ -39,8 +39,31 @@
 
 # Make a new player object that is currently in the 'outside' room.
 
+player = Player("Adnan", room['outside'].name)
+
 # Write a loop that:
-#
+
+
+def adventure_game():
+    print("Welcome to the Adventure Game")
+    print(f"Welcome {player.name}!")
+    print(f"Your Current Room is {player.current_room}")
+
+    while True:
+        command = input("Enter commands --> n, e, s, w OR q to quit: ")
+
+        print(f"You have entered {command!r}")
+
+        if command == 'n':
+            print("Lets go the North Room")
+        elif command == 'q':
+            print("Exiting game")
+            break
+
+
+if __name__ == '__main__':
+    adventure_game()
```

Comment:
Nice job finding and using this. It's a good pattern that allows a file to be both imported and run directly.